<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>580</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>Arm Controls</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QLabel" name="lbl_motoSA">
               <property name="text">
                <string>Shoulder A: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDial" name="dial_MotorSA">
               <property name="maximum">
                <number>180</number>
               </property>
               <property name="sliderPosition">
                <number>90</number>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rBtn_MtrSA">
               <property name="text">
                <string>Export</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_2">
             <item>
              <widget class="QLabel" name="lbl_motorSB">
               <property name="text">
                <string>Shoulder B:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDial" name="dial_MotorSB">
               <property name="maximum">
                <number>180</number>
               </property>
               <property name="sliderPosition">
                <number>90</number>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rBtn_MtrSB">
               <property name="text">
                <string>Export</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <item>
              <widget class="QLabel" name="lbl_motorElbow">
               <property name="text">
                <string>Elbow:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDial" name="dial_MotorElbow">
               <property name="maximum">
                <number>180</number>
               </property>
               <property name="sliderPosition">
                <number>90</number>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rBtn_MtrElbow">
               <property name="text">
                <string>Export</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_4">
             <item>
              <widget class="QLabel" name="lbl_motorClamp">
               <property name="text">
                <string>Clamp:</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDial" name="dial_MotorClamp">
               <property name="maximum">
                <number>30</number>
               </property>
               <property name="value">
                <number>15</number>
               </property>
               <property name="sliderPosition">
                <number>15</number>
               </property>
               <property name="invertedAppearance">
                <bool>false</bool>
               </property>
               <property name="invertedControls">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QRadioButton" name="rBtn_MtrClaw">
               <property name="text">
                <string>Export</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QSlider" name="vSdr_speed">
               <property name="minimum">
                <number>5</number>
               </property>
               <property name="maximum">
                <number>15</number>
               </property>
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="invertedAppearance">
                <bool>true</bool>
               </property>
               <property name="invertedControls">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Fast</string>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="verticalSpacer">
                 <property name="orientation">
                  <enum>Qt::Vertical</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>20</width>
                   <height>40</height>
                  </size>
                 </property>
                </spacer>
               </item>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Slow</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QRadioButton" name="rBtn_MtrNone">
             <property name="text">
              <string>None</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QCustomPlot" name="temp_graph" native="true">
        <property name="minimumSize">
         <size>
          <width>421</width>
          <height>0</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_9">
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Actions</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_8">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_7">
             <item>
              <widget class="QPushButton" name="pBtn_reset">
               <property name="text">
                <string>Reset</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pBtn_close">
               <property name="text">
                <string>Close</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="pBtn_setinitPos">
               <property name="text">
                <string>Set Initial Pos</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QDoubleSpinBox" name="dsBox_tempLimit">
               <property name="prefix">
                <string/>
               </property>
               <property name="suffix">
                <string>C</string>
               </property>
               <property name="singleStep">
                <double>0.100000000000000</double>
               </property>
               <property name="value">
                <double>50.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <spacer name="verticalSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="lbl_actualTemp">
             <property name="text">
              <string>Temp :</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lbl_actualPressure">
             <property name="text">
              <string>Pressure :</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>580</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
